src/main/scala/org/downtowndailybread/ApiGlobalResources.scala

```diff
@@ -0,0 +1,19 @@
+package org.downtowndailybread
+
+import akka.http.scaladsl.model.Uri
+import akka.http.scaladsl.server.Directives._
+
+trait ApiGlobalResources {
+
+  val version = "v1"
+
+  val port = 8090
+
+
+  def generateUri(uriEnding: String): Uri = {
+    Uri.apply(s"api/${version}/$uriEnding")
```

Comment:
Why is the string interpolation different for `version` and `uriEnding`?

Reply:
generateUri is actually not used anymore. I'll remove.